charts/timed/values.yaml

```
@@ -307,13 +305,19 @@ redmine:
   url: redmine.example.com
   # redmine.apiKey -- Redmine API Key
   apiKey: ""
-  # redmine.htaccessUser -- Redmine htaccess user
-  htaccessUser: ""
-  # redmine.htaccessPassword -- Redmine htaccess password
-  htaccessPassword: ""
   # redmine.spenthoursField -- Spent hours field id on Redmine
   spenthoursField: 0
 
+sentry:
```

The sentry settings are specifig to the backend:
```suggestion
backend:
  sentry:
```

---

Hmm there's more settings like `redmine` which is its own object, so I figured to keep it separate too.
I can integrate them both into `backend.settings`, sure

---

It matters here because we might also add an integration for the frontend and that will need it's own DSN. redmine and others don't have this distinction.